### ISSUE TITLE
Extra arrows showing

### ISSUE THREAD

**Issue by s10wen** [s10wen/SO-Dark-Monokai-v3#17]
https://github.com/simonowendesign/SO-Dark-Monokai-v3/commit/68994c2e734fcecd4530cf6c6f4c6e10c3243563

**Comment by s10wen** [comment]
![screen shot 2013-06-25 at 14 48 46](https://f.cloud.github.com/assets/1136114/702846/f9617b9c-dd9d-11e2-8e15-f2066907d969.png)

**Comment by RaphaelDDL** [comment]
On v28 @ Win7
![chrome28](https://f.cloud.github.com/assets/1087109/703406/d75172bc-dda9-11e2-8ae2-4c9847a8a8ac.png)
(No extra arrow, but arrow is black with border)

**Comment by s10wen** [comment]
![screen shot 2013-06-25 at 18 12 15](https://f.cloud.github.com/assets/1136114/704184/cede105c-ddba-11e2-8eb8-99cc9b6f0835.png)

**Comment by RaphaelDDL** [comment]
I applied the patch, seems nothing changed here. Or it's v28 or Windows specific (since seems like you are using Mac from the way the fonts are in your prints).

I tried go on `chrome-devtools://devtools/devTools.css` on v28 to check the selectors and maybe help you but returns this

![chromev28](https://f.cloud.github.com/assets/1087109/705298/e5568c7e-ddd3-11e2-9206-6db41452169e.png)

Any idea?

Edit:
I opened the `chrome-devtools://devtools/devtools.html` and seems the only external stylesheet that it calls is `inspector.css` ( `chrome-devtools://devtools/inspector.css` ) instead of `devTools.css`.
